### PR TITLE
support to set `dbus/vendored`

### PR DESCRIPTION
**Pull request by mokurin000**

### CODE REVIEW

**Comment by Mange** [comment]
Please explain what this is supposed to accomplish. :-)

**Comment by mokurin000** [comment]
> Please explain what this is supposed to accomplish. :-)

without this patch, users of `mpris-rs` will need to add `dbus` to their dependencies to set "dbus/vendored"

or they could simply intro "mpris/vendored" to make dbus build vendored

**Comment by Mange** [comment]
`dbus-rs` defines this feature as such:

> The `vendored` feature links libdbus statically into the final executable.

Seems useful, yes.

File: Cargo.toml [hunk 1]
```diff
@@ -26,3 +26,7 @@ thiserror = "1.0.37"
 [dev-dependencies]
 anyhow = "1.0.66"
 termion = "2.0.1"
+
+[features]
+default = []
+vendored = ["dbus/vendored"]
```
Comment (Mange) [inline Cargo.toml]: Let's rename the feature to `dbus-vendored`.

**Review by Mange — CHANGES_REQUESTED** [review]

**Comment by Mange** [comment]
Thank you!

**Pull request merged**